firmware/bluetooth_rxtx/bluetooth_rxtx.c

```diff
@@ -1800,10 +1483,10 @@ void bt_le_sync(u8 active_mode)
 
 		const uint32_t *whit = whitening_word[btle_channel_index(channel-2402)];
 		for (i = 0; i < 4; i+= 4) {
-			uint32_t v = rxbuf1[i+0] << 24
-					   | rxbuf1[i+1] << 16
-					   | rxbuf1[i+2] << 8
-					   | rxbuf1[i+3] << 0;
+			uint32_t v = active_rxbuf[i+0] << 24
+					   | active_rxbuf[i+1] << 16
+					   | active_rxbuf[i+2] << 8
+					   | active_rxbuf[i+3] << 0;
```

Comment:
This code was intentionally using rxbuf1. It most likely will not work correctly using active_rxbuf